themes/classic/assets/js/theme.js

```diff
@@ -150,7 +150,7 @@
 /***/ function(module, exports, __webpack_require__) {
 
 	var __WEBPACK_AMD_DEFINE_ARRAY__, __WEBPACK_AMD_DEFINE_RESULT__;/*!
-	 * jQuery JavaScript Library v2.2.3
+	 * jQuery JavaScript Library v2.2.1
```

Comment:
Please, run a `npm update` and re-build because you're downgrading jQuery version 😉